### PR TITLE
[hotfix] Fix links to old documentation

### PR DESCRIPTION
**Pull request by alibahadirzeybek**

### CODE REVIEW

File: long-ride-alerts/README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The challenge is figuring out what state and timers to use, and when to set and
 ## Documentation
 
 - [ProcessFunction](https://ci.apache.org/projects/flink/flink-docs-stable/dev/stream/operators/process_function.html)
-- [Working with State](https://ci.apache.org/projects/flink/flink-docs-stable/dev/stream/state/index.html)
+- [Working with State](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/fault-tolerance/state)
```
Comment (NicoK) [inline long-ride-alerts/README.md:80]: This link should point to flink-docs-stable, not master

File: rides-and-fares/README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ For the purposes of this exercise it's okay to assume that the START and fare ev
 
 ## Documentation
 
-- [Working with State](https://ci.apache.org/projects/flink/flink-docs-stable/dev/stream/state/index.html)
+- [Working with State](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/fault-tolerance/state)
```
Comment (NicoK) [inline rides-and-fares/README.md:80]: same here...

**Review by NicoK — CHANGES_REQUESTED** [review]

File: long-ride-alerts/README.md (hunk 1 above)
Comment (alibahadirzeybek) [reply long-ride-alerts/README.md]: Fixed

File: rides-and-fares/README.md (hunk 2 above)
Comment (alibahadirzeybek) [reply rides-and-fares/README.md]: Fixed

**Comment by alibahadirzeybek** [comment]
Changed other links to old documentation

**Review by NicoK — APPROVED** [review]
Thanks, lgtm.

I'll also merge this to https://github.com/apache/flink-training

**Pull request merged**